index.js

```diff
@@ -40,44 +40,48 @@ I18nPlugin.prototype.apply = function(compiler) {
 		compilation.dependencyFactories.set(ConstDependency, new NullFactory());
 		compilation.dependencyTemplates.set(ConstDependency, new ConstDependency.Template());
 	});
-	compiler.parser.plugin("call " + this.functionName, function(expr) {
-		var param, defaultValue;
-		switch(expr.arguments.length) {
-		case 2:
-			param = this.evaluateExpression(expr.arguments[1]);
-			if(!param.isString()) return;
-			param = param.string;
-			defaultValue = this.evaluateExpression(expr.arguments[0]);
-			if(!defaultValue.isString()) return;
-			defaultValue = defaultValue.string;
-			break;
-		case 1:
-			param = this.evaluateExpression(expr.arguments[0]);
-			if(!param.isString()) return;
-			defaultValue = param = param.string;
-			break;
-		default:
-			return;
-		}
-		var result = localization ? localization(param) : defaultValue;
-		if(typeof result == "undefined") {
-			var error = this.state.module[__dirname];
-			if(!error) {
-				error = this.state.module[__dirname] = new MissingLocalizationError(this.state.module, param, defaultValue);
-				if (failOnMissing) {
-					this.state.module.errors.push(error);
-				} else {
-					this.state.module.warnings.push(error);
+	compiler.plugin("compilation", function(compilation, data) {
```

Comment:
Thanks for checking that. If there's a way to check wether webpack 1 or 2 is running, we'd be happy to add that. Otherwise I guess for time being this PR should be closed.